### ISSUE TITLE
export other interfaces besides `Chart`

### ISSUE THREAD

**Issue by tomwayson** [Esri/cedar#391]
We don't currently export the other interfaces (`IDataset`, `ISeries`, etc). See:

https://unpkg.com/@esri/cedar@1.0.0-beta.2/dist/esm/index.d.ts

This https://github.com/Esri/cedar/blob/master/packages/cedar/src/index.ts#L1

Should be `export * from ./Chart.ts` - or the interfaces need to move into a shared package (if needed in cedar-amcharts)~~, or published to DefinitelyTyped~~, or whatever.

**Comment by tomwayson** [comment]
Turns out, [If your package is written in TypeScript then you should publish the types w/ your package](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html).

**Comment by tomwayson** [comment]
This is causing a bug for me in a TS app. I expect to be able to import ES modules and types like:

```ts
import { Chart, IDefinition } from '@esri/cedar';
```

Instead I had to:

```ts
import Chart, { IDefinition } from '@esri/cedar/dist/esm/Chart';
```

My goal when I set up the exports was to for UMD builds to expose a `cedar` global that everything hangs off of, but for consumers of the ESM modules to be able to import from `@esri/cedar`. I didn't really know what I was doing, which is why I made a [default export](https://github.com/Esri/cedar/blob/db41632fcfef5fd7b029a3a84a835d412bfb8ff6/packages/cedar/src/index.ts#L4) and started hanging things off of it.

Instead I think what we need to do is remove the default export and `export * from './all/the/Things'` and then configure rollup to build the global from _named_ exports, instead of the default export.
